Update the "JupyterLab and notebook interface" episode

bast opened the pull request:
- remove outdated/wrong info
- reorder
- more readable by avoiding super-wide tables which require side-scrolling
- better section headings
- better code-highlighting

bast committed:
we don't talk about version control in this episode

bast committed:
code-highlight markdown example

bast committed:
move "components" up, this has nothing to do with "cells"

bast committed:
no, we don't talk about Git integration here

bast committed:
better section heading

bast committed:
table -> bullet list

I don't think the table is helpful here for long sentence cells

bast committed:
remove "output" which I find redundant here

bast committed:
split 1 too wide table into 2 more readable tables

annefou commented on content/interface.md:
````diff
@@ -1,30 +1,40 @@
 $ cd jupyterlab-demo
 $ jupyter-lab
 ```
 - On Windows, the JupyterLab App can also be launched by clicking on the JupyterLab icon in the Anaconda menu.
````
according they have installed it. I don't think I have it with miniconda3.

bast replied:
Ah good catch.

bast committed:
rm sentence about JupyterLab icon in the Anaconda menu

since this only applies to Anaconda but not how we asked participants
to install software for our workshop
